documentation/book/con-accessing-kafka-bridge-from-outside.adoc

```diff
@@ -6,7 +6,7 @@
 
 = Accessing the Kafka Bridge outside of Kubernetes
 
-After deployment, the {ProductName} Kafka Bridge can only be accessed by applications running in the same Kubernetes cluster. These applications use the `_kafka-bridge-name_-bridge-service` Service to access the API.
+After deployment, the {ProductName} Kafka Bridge can only be accessed by applications running in the same Kubernetes cluster. These applications use the `{kafka-bridge-name}-bridge-service` Service to access the API.
```

Comment:
Do we use `{...}` for replaceables now? It used to be `<...>` in the past.

Reply:
I've reverted back to angled brackets throughout, except in references to the API methods, such as  `/topics/{topicname}`. We should keep the curly brackets there to match the Bridge API reference documentation.